plugins/include/float.inc

```diff
@@ -13,6 +13,9 @@
 #endif
 #define _float_included
 
+/* !!! Don't ever touch !!! */
```

Comment:
Well, yeah, I would prefer to see a more serious comment, with proper explanation and comment format.